src/test/resources/pages/instructorArchiveStatusNotAffected.html

```diff
@@ -34,86 +34,92 @@
+                     Action(s)
+                  </th>
+               </tr>
+            </thead>
+            <tbody>
+               <tr>
+                  <td id="courseid0">
+                     CCAddUiTest.CS1101
+                  </td>
+                  <td id="coursename0">
+                     Programming Methodology
+                  </td>
+                  <td class="align-center">
+                     0
+                  </td>
+                  <td class="t_course_teams align-center">
+                     0
+                  </td>
+                  <td class="align-center">
+                     0
+                  </td>
+                  <td class="align-center">
+                     0
+                  </td>
+                  <td class="align-center no-print">
+                     <a class="btn btn-default btn-xs t_course_enroll0" data-original-title="Enroll student into the course" data-placement="top" data-toggle="tooltip" href="/page/instructorCourseEnrollPage?courseid=CCAddUiTest.CS1101&user=CCAddUiTest.instr3" title="">
+                        Enroll
+                     </a>
+                     <a class="btn btn-default btn-xs t_course_view0" data-original-title="View, edit and send invitation emails to the students in the course" data-placement="top" data-toggle="tooltip" href="/page/instructorCourseDetailsPage?courseid=CCAddUiTest.CS1101&user=CCAddUiTest.instr3" title="">
+                        View
+                     </a>
+                     <a class="btn btn-default btn-xs t_course_edit0" data-original-title="Edit Course information and instructor list" data-placement="top" data-toggle="tooltip" href="/page/instructorCourseEditPage?courseid=CCAddUiTest.CS1101&user=CCAddUiTest.instr3" title="">
+                        Edit
+                     </a>
+                     <a class="btn btn-default btn-xs t_course_archive0" data-original-title="Archive the course so that it will not be shown in the home page any more (you can still access it from the 'Courses' tab)" data-placement="top" data-toggle="tooltip" href="/page/instructorCourseArchive?courseid=CCAddUiTest.CS1101&archive=true&next=%2Fpage%2FinstructorCoursesPage&user=CCAddUiTest.instr3" title="">
+                        Archive
+                     </a>
+                     <a class="btn btn-default btn-xs t_course_delete0" data-original-title="Delete the course and its corresponding students and sessions" data-placement="top" data-toggle="tooltip" href="/page/instructorCourseDelete?courseid=CCAddUiTest.CS1101&next=%2Fpage%2FinstructorCoursesPage&user=CCAddUiTest.instr3" onclick="return toggleDeleteCourseConfirmation('CCAddUiTest.CS1101');" title="">
+                        Delete
+                     </a>
+                  </td>
+               </tr>
+            </tbody>
+         </table>
+         <br/>
+                  <br/>
+                  <br/>
+                  <br/>
+               </div>
+   </div>
 
```

Comment:
It's fine if the change is actually required as in this case. Hopefully, that situation will not be frequent.